<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pinUI</class>
 <widget class="QDialog" name="pinUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="infoScreen">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item row="1" column="2">
          <widget class="QPushButton" name="button6">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="button0">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="button5">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="button3">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="button1">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="button4">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="button8">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="button7">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="button2">
           <property name="font">
            <font>
             <family>Franklin Gothic Heavy</family>
            </font>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="button9">
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="buttonClr">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBck">
           <property name="text">
            <string>&lt;--</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonEnter">
           <property name="text">
            <string>Enter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
